website/docs/guides/qwik.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Qwik
+section: Frameworks
+---
+
+# Qwik
```

Comment:
This should be `##` instead of `#` for the table of contents sidebar on the right.